engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/SuggestionsHandlerEventsTest.scala

```diff
@@ -114,29 +114,7 @@ class SuggestionsHandlerEventsTest extends BaseServerTest with FlakySpec {
                     }
                   ],
                   "returnType" : "MyAtom",
-                  "documentation" : " PRIVATE\n\n A key-value store. This type assumes all keys are pairwise comparable,\n using the `<`, `>` and `==` operators.\n\n Arguments:\n - one: The first.\n - two_three: The *second*.\n\n ? Info\n   Here is a thing.",
-                  "documentationSections" : [
-                    {
-                      "type" : "tag",
-                      "name" : "PRIVATE",
-                      "body" : ""
-                    },
-                    {
-                      "type" : "paragraph",
-                      "body" : "A key-value store. This type assumes all keys are pairwise comparable, using the <code>&lt;</code>, <code>&gt;</code> and <code>==</code> operators. "
-                    },
-                    {
-                      "type" : "keyed",
-                      "key" : "Arguments",
-                      "body" : " <ul><li>one: The first.</li><li>two_three: The <b>second</b>.</li></ul> "
-                    },
-                    {
-                      "type" : "marked",
-                      "mark" : "Info",
-                      "header" : "Info",
-                      "body" : " Here is a thing."
-                    }
-                  ]
+                  "documentation" : " PRIVATE\n\n A key-value store. This type assumes all keys are pairwise comparable,\n using the `<`, `>` and `==` operators.\n\n Arguments:\n - one: The first.\n - two_three: The *second*.\n\n ? Info\n   Here is a thing."
```

Comment:
OK. No more `documentationSections` being sent. Just the plain `documentation` string.